### ISSUE TITLE
Add Generated AUTH_API_KEY to .env file during seeds.exs (setup)

### ISSUE THREAD

**Issue by nelsonic** [dwyl/auth#87]
At present we have code that generates a New "Admin" `AUTH_API_KEY` in the `seeds.exs` script:

https://github.com/dwyl/auth/blob/fafc5e059d65de420b4d99bf1d15ac45a3bcda12/priv/repo/seeds.exs#L36-L53

But we are having to _manually_ copy this environment variable ... it's getting old.

# Todo

+ [x] Append the `AUTH_API_KEY` `.env` file and then `source .env` during execution of `seeds.exs`

**Comment by nelsonic** [comment]
Having fun with this error:
```elixir
** (FunctionClauseError) no function clause matching in Enum."-join/2-lists^foldl/2-0-"/3
```
😕

**Comment by nelsonic** [comment]
This is included in PR #85
